metrics/Contributing_Organizations.md

```diff
@@ -3,27 +3,24 @@
 Question: What is the number of contributing organizations?
 
 ## Description
-A description of what the metric is and what it captures.
-The first few sentences should match the description in the [metrics list](../activity-metrics-list.md).
-
+These are the contributions made by organizations that are seperate from the project being assessed. This does not include individual contributors to the project. For example, if a software company such as Google commited changes, it would count twards the number of contributing organizations, but an individual employee of Google would not count as being seperate. 

```

Comment:
This is a more granular variation on Elephant Factor. I think the purpose is to count the total number of organizations who have made contributions.